### PR TITLE
[Enchancement] Date and Venue redirected

### PR DESCRIPTION
**Pull request by vipulgupta2048**
The date now redirects to make a new calendar entry in Google Calendar for the user.
Venue redirects to maps API in the map section of the website.

### CODE REVIEW

**Review by realslimshanky — CHANGES_REQUESTED** [review]
You need to make changes to `templates/index.html` and then build using static jinja instead of editing directly to the generated file.

**Comment by vipulgupta2048** [comment]
Rookie mistake, my bad. Won't happen again. 
Well, done now. Learnt something new. 
Also I am unsure of as to how there are changes in index.html now. I didn't even open that file. I didn't change anything there. Yet everytime I did git diff, it showed changes.

File: index.html [hunk 1]
```diff
@@ -2512,7 +2483,7 @@ <h3 class="wow fadeInDown">Silver Sponsors</h3>
       <ul class="tg-sponsors col-md-4">
         <li>
           <figure class="tg-sponder">
-            <a data-toggle="modal" href="#zeomega-modal" target="_blank" href="http://www.zeomega.com/">
+            <a href="http://www.zeomega.com/" target="_blank">
```
Comment (CuriousLearner) [inline index.html:2486]: This is supposed to open up a modal. In the modal, you can link their website which can be opened up in another tab.

**Review by CuriousLearner — CHANGES_REQUESTED** [review]
Please see the review inline. 

Also for generating actual pages from templates, please use static jinja (Read the `README`)

File: index.html (hunk 1 above)
Comment (CuriousLearner) [reply index.html:2486]: So, revert the code where you've removed the modal code :) and do the changes for the link in the modal itself.

**Comment by realslimshanky** [comment]
@vipulgupta2048 you can close this PR since you have created a new one i.e. #124

**Pull request closed without merging**